src/packages/core/workspace/components/workspace-action/default/workspace-action.element.ts

```diff
@@ -1,8 +1,8 @@
-import type { UmbWorkspaceAction } from '../../../../index.js';
 import { UmbActionExecutedEvent } from '@umbraco-cms/backoffice/event';
 import { html, customElement, property, state } from '@umbraco-cms/backoffice/external/lit';
 import type { UUIButtonState } from '@umbraco-cms/backoffice/external/uui';
 import { UmbLitElement } from '@umbraco-cms/backoffice/lit-element';
+import type { UmbWorkspaceAction } from '@umbraco-cms/backoffice/workspace';
```

Comment:
Doesn't this import from itself creating a cyclic dependency?

Reply:
Sharp one, I couldn't make it complain about it, I guess its because it's an type import(only a compile time import) and therefor okay with circular. I have of-course changed it to prevent a potential issue here :-)

Reply:
You are right, the `import type` prevents it from being an actual runtime danger :)